plugins/action/k8s_info.py

```diff
@@ -71,6 +75,118 @@ def run(self, tmp=None, task_vars=None):
         if src:
             new_module_args['src'] = src
 
+        template = self._task.args.get('template', None)
+        if template:
+            # template is only supported by k8s module.
+            if self._task.action not in ('k8s', 'community.kubernetes.k8s', 'community.okd.k8s'):
```

Comment:
Hmm, I still am not a fan of naming specific modules here (and referencing a module in another collection seems odd), is there not a way to do something like verify the `template` parameter is in the argspec of the module?

Reply:
@fabianvf that's a fair point, thank you for raising it :+1:

Reply:
@fabianvf I am afraid that there is no way to do that. Since the Action plugin runs before argspec validation which is in the module code. Ansible core repos also do this kind of validation https://github.com/ansible/ansible/pull/71824/files#diff-b77962b6b54a830ec373de0602918318R86 (referring modules from other collections).

Reply:
hmm, if there's not an easy way to do it in a more generic way then I'm ok with merging as-is

Reply:
OK.